packages/hardhat-core/README.md

```diff
@@ -34,7 +34,7 @@ On [Hardhat's website](https://hardhat.org) you will find:
 
 Contributions are always welcome! Feel free to open any issue or send a pull request.
 
-Go to [CONTRIBUTING.md](./CONTRIBUTING.md) to learn about how to set up Hardhat's development environment.
+Go to [CONTRIBUTING.md](https://github.com/NomicFoundation/hardhat/blob/main/CONTRIBUTING.md) to learn about how to set up Hardhat's development environment.
```

Comment:
This one makes more sense. The relative path only works with the symlink in the root of the repo, but not in -core